brave/brave-core
# Add wallet button counter

Notification badge with counter added for pending transactions
![image](https://user-images.githubusercontent.com/106654560/208529714-5332c21b-9902-4d24-a56d-108677f7855a.png)
![image](https://user-images.githubusercontent.com/106654560/208529730-85233264-854a-4736-9bfb-2c70d71c5786.png)

Also notification badge without counter is shown for users that don't have wallet yet.

Resolves https://github.com/brave/brave-browser/issues/24818


Resolves 

## Submitter Checklist:

  * `npm run test -- brave_browser_tests`, `npm run test -- brave_unit_tests` [wiki](https://github.com/brave/brave-browser/wiki/Tests)
  * `npm run lint`, `npm run presubmit` [wiki](https://github.com/brave/brave-browser/wiki/Presubmit-checks), `npm run gn_check`, `npm run tslint`

## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on

## Test Plan:
1) Create some pending transactions in the wallet.
2) Check counter
3) Approve some of them
4) Check that count is correct and bubble disappears after all transactions are confirmed
5) Check that counters for brave rewards button and brave shields button are correctly shown
